python/datashaper/datashaper/engine/verbs/unroll.py

```diff
@@ -3,10 +3,13 @@
 # Licensed under the MIT license. See LICENSE file in the project.
 #
 
+from datashaper.engine.verbs.verbs_mapping import verb

```

Comment:
I think all of these can be `from .verbs_mapping import verb`

Reply:
I reallllly don't like relative imports xD maybe it's a personal thing, but after discussing with Rodrigo, in other projects, we usually do relative imports to resolve circular imports and in the `__init__.py` files and on the other places use absolute imports, what do you think? I'm working on doing this for datashaper project already.

Reply:
+1 I personally prefer absolute imports. Relative imports are very obscure imo, in these things I prefer verbosity, to know exactly what I'm importing without needing to inspect.